bitlbee.h

```diff
@@ -121,7 +121,7 @@ extern "C" {
 #define CONTROL_TOPIC "Welcome to the control channel. Type \2help\2 for help information."
 #define IRCD_INFO PACKAGE " <http://www.bitlbee.org/>"
 
-#define MAX_NICK_LENGTH 24
+#define MAX_NICK_LENGTH 80
```

Comment:
I think you forgot to get drop this part of the diff. It's probably safe to change it locally for yourself but merging it has more implications.